### PR TITLE
Add initial readme

### LINKED ISSUE #1: Missing README
No README for this library makes it hard to use for someone that doesn't know what it does or how it works

### PR DESCRIPTION
**Pull request by kerrmarin**
Fixes #1  & #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+- Clone this repo to your local machine using `https://github.com/design-ops/sketch-to-swiftUI`
+
+### Setup
+
+> Install dependencies first
+
+```shell
+$ brew install fonttools
+```
+
```
Comment (deanWombourne) [inline README.md:22]: Do you want to add a note that this is optional, and only required if the fonts found happen to be in a font collection?

Comment (atomoil) [reply README.md:22]: 👍

File: README.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+# Sketch to SwiftUI
+
+> Create a StylableSwiftUI stylist implementation from a Sketch file. Sketch -> SwiftUI
+
+## Installation
+
+- Clone this repository
+- Run `npm i`
```
Comment (atomoil) [inline README.md:9]: Hmm, I/we should push to npm then we can just say run `npm i -g sketch-to-swiftUI` and then `sketch-to-swiftUI <file>`

File: README.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+### Step 3
+
+- 🔃 Create a new pull request using <a href="https://github.com/design-ops/sketch-to-swiftui/compare/" target="_blank">`https://github.com/design-ops/sketch-to-swiftui/compare/`</a>.
+
+---
+
+## License
+
+See the LICENSE file.
```
Comment (atomoil) [inline README.md:72]: There is no LICENSE file 😳 - should be MIT tho'

**Pull request merged**